Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM alpine:3
+FROM golang:alpine3.19
 
 RUN apk add --no-cache \
-        curl
+        git
 
-RUN mkdir -p /opt/buildevents
+RUN git clone \
```

Comment:
No need to clone. You can just go install.

Reply:
Hugh it looks like we are running into this https://github.com/golang/go/issues/44840